ledger-tool/src/main.rs

```diff
@@ -4141,9 +4147,30 @@ fn main() {
                     force_update_to_open,
                 );
                 let num_slots = value_t_or_exit!(arg_matches, "num_slots", usize);
-                let slots = blockstore
-                    .get_latest_optimistic_slots(num_slots)
-                    .expect("Failed to get latest optimistic slots");
+                let exclude_vote_only_slots = arg_matches.is_present("exclude_vote_only_slots");
```

Comment:
wdyt about flipping this logic?  is the subcommand used frequently enough that doing so would break peoples' workflow?

Reply:
This is a command that is new with v1.14, so debatably there may not be any workflows depending on it yet. I can't recall, do we claim to make any guarantees about compatibility of a branch before it has gone to mainnet?

Reply:
But in general, I'd be fine in flipping the logic. If I recall correctly, the original intent of storing these slots / exposing `ledger-tool` command was to assist in restarts. Is that right @jbiseda ?

Reply:
@steviez Yes, that's correct. The idea was to have a somewhat cleaner way of finding latest optimistic slots for restart than relying on logs.

Reply:
Updated, and update the help to call out that we exclude vote-only slots by default